src/protocol/requests/deleteTopics/v0/response.js

```diff
@@ -0,0 +1,37 @@
+const Decoder = require('../../../decoder')
+const { failure, createErrorFromCode } = require('../../../error')
+
+/**
+ * DeleteTopics Response (Version: 0) => [topic_error_codes]
+ *   topic_error_codes => topic error_code
+ *     topic => STRING
+ *     error_code => INT16
+ */
+
+const topicNameComparator = (a, b) => a.topic.localeCompare(b.topic)
+
+const topicErrors = decoder => ({
+  topic: decoder.readString(),
+  errorCode: decoder.readInt16(),
+})
+
+const decode = async rawData => {
+  const decoder = new Decoder(rawData)
+  return {
+    topicErrors: decoder.readArray(topicErrors).sort(topicNameComparator),
+  }
+}
+
+const parse = async data => {
+  const topicsWithError = data.topicErrors.filter(({ errorCode }) => failure(errorCode))
+  if (topicsWithError.length > 0) {
```

Comment:
OK, I had some more thought to this problem, and I don't want to solve it in this PR.
It will require a new type of error and some rework of the network retry mechanism, which should check each error and group the ones that can be retried (similar to the producer API). We might never implement this as a generic feature, so I want to unblock this PR.

Reply:
Fair enough. Will you create an issue to cover this so that it doesn't get lost?